### ISSUE TITLE
Option to log to console instead of file when in interactive mode?

### ISSUE THREAD

**Issue by Artalus** [jfrog/jfrog-cli#1194]
```
$ jfrog rt upload '(x|x)' test-repo/test-folder/
 Log path: .../.jfrog/logs/jfrog-cli.2021-08-10.15-49-06.904585.log
{
  "status": "failure",
  "totals": {
    "success": 0,
    "failure": 0
  }
}
```
This is rather verbose and completely useless output for me regular terminal pleb, experimenting with new CLI toy. To see the actual error of said toy, I have to look not at stderr, but at a file for some reason:
```
$ cat ~/.jfrog/logs/jfrog-cli.2021-08-10.15-49-06.904585.log 
[Error] Path does not exist: (x|x)
[Error] Path does not exist: (x|x)
```
Apparently, if I set `export CI=true`, then I can see them logs immediately - but then I miss various interactive stuff.

Is it possible to see the logs in terminal without pretending to be CI, as well as having to copy-paste paths to files for `cat`?

**Comment by scentoni** [comment]
+1. If anything, the behavior of `jfrog` with respect to the `CI` variable is precisely backwards. In interactive use, commands should _always_ provide their error output to stderr. In a scripted CI workflow, it may sometimes be more helpful to capture error logs in a file for parsing, delivering, and storage.

**Comment by venkatanrv** [comment]
Hi @Artalus @scentoni 
Thanks for raising this point.

Thank you for bringing this question, in order to print the detailed  logs in the terminal, you can use the variable “export JFROG_CLI_LOG_LEVEL=DEBUG” for reading or parsing CLI output. I hope this helps

Attaching the documentation link for your reference: https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli/usage

**Comment by scentoni** [comment]
@venkatanrv No, the variable `JFROG_CLI_LOG_LEVEL` is not relevant to this question. The variable `CI` is relevant, as described above.

**Comment by RemiBou** [comment]
Closing as not planned. Either use CI=true and get everything in stdout or use interactive mode and read the file.
